src/jade/tabs/contacts.jade

```diff
@@ -99,7 +99,7 @@ section.col-xs-12.content(ng-controller="ContactsCtrl")
       span.name(ng-hide='editing', ng-bind="entry.name") 
     .col-xs-12.col-sm-6.col-md-6(ng-hide='editing')
       div
-        b.hidden-xs {{entry.address | rpaddressorigin}}: {{entry.address}}
+        b.hidden-xs {{entry.address | rpaddressorigin}}: {{contactNames[entry.address] || resolveName(entry.address)}}
```

Comment:
Still not clear. If it always return ```entry.address``` back, then why bother doing the ```id.resolveName``` at all here? I think you wanted to anticipate the case when ```entry.address``` is resolvable but yet not in ```contactNames```. But this is not necessary.